cmake/faceliftMacros.cmake

```diff
@@ -388,6 +390,26 @@ macro(_facelift_parse_target_arguments additionalOptions additionalOneValueArgs
         list(APPEND HEADERS ${GLOB_FILES})
     endforeach()
 
+    set(HEADERS_NO_MOC ${ARGUMENT_HEADERS_NO_MOC})
+
+    # files for installation can be in both sections: HEADERS and HEADERS_NO_MOC
```

Comment:
I see no reason why having a header in both sections should be allowed.
We should define one proper way of doing things.

Reply:
changed, added a check.
let me know if there is not too much inform messages
I've added them only to make the error clear to the user